modules/tinymce/src/core/main/ts/api/dom/RangeUtils.ts

```diff
@@ -71,34 +71,32 @@ function RangeUtils(dom: DOMUtils): RangeUtils {
     split,
     normalize
   };
-}
+};
 
-namespace RangeUtils {
-  /**
-   * Compares two ranges and checks if they are equal.
-   *
-   * @static
-   * @method compareRanges
-   * @param {DOMRange} rng1 First range to compare.
-   * @param {DOMRange} rng2 First range to compare.
-   * @return {Boolean} true/false if the ranges are equal.
-   */
-  export const compareRanges = RangeCompare.isEq;
+/**
+ * Compares two ranges and checks if they are equal.
+ *
+ * @static
+ * @method compareRanges
+ * @param {DOMRange} rng1 First range to compare.
+ * @param {DOMRange} rng2 First range to compare.
+ * @return {Boolean} true/false if the ranges are equal.
+ */
+RangeUtils.compareRanges = RangeCompare.isEq;
 
-  /**
-   * Gets the caret range for the given x/y location.
-   *
-   * @static
-   * @method getCaretRangeFromPoint
-   * @param {Number} clientX X coordinate for range
-   * @param {Number} clientY Y coordinate for range
-   * @param {Document} doc Document that x/y are relative to
-   * @returns {Range} caret range
-   */
-  export const getCaretRangeFromPoint = CaretRangeFromPoint.fromPoint as (clientX: number, clientY: number, doc: Document) => Range;
+/**
+ * Gets the caret range for the given x/y location.
+ *
+ * @static
+ * @method getCaretRangeFromPoint
+ * @param {Number} clientX X coordinate for range
+ * @param {Number} clientY Y coordinate for range
+ * @param {Document} doc Document that x/y are relative to
+ * @returns {Range} caret range
+ */
+RangeUtils.getCaretRangeFromPoint = CaretRangeFromPoint.fromPoint as (clientX: number, clientY: number, doc: Document) => Range;
```

Comment:
Was wondering if the type assertions are required here?

Reply:
It was like this previously, so I'm hesitant to remove it. It shouldn't be needed though...

Reply:
Removed the cast as discussed on slack